content/shmem_malloc.tex

```diff
@@ -79,8 +87,8 @@
     The \FUNC{shmem\_realloc} routine returns a pointer to the allocated space
     (which may have moved); otherwise, it returns a null pointer.
     
-    The \FUNC{shmem\_align} routine returns an aligned pointer to the allocated
-    space; otherwise, it returns a null pointer.
+    The \FUNC{shmem\_align} routine returns an aligned pointer whose value is a
+    multiple of \VAR{alignment}; otherwise, it returns a null pointer.
```

Comment:
Split this into 4 sections.

Reply:
Can somebody help with my poor memory --- Was this a note to split the current `shmem_malloc` section into separate `shmem_malloc`, `shmem_free`, `shmem_realloc`, and `shmem_align` sections?  Sorry, I should have taken better notes.

Reply:
I remember the same -- split the `shmem_malloc` section into four new sections for each function.

Reply:
👍  I remember the same.